operator/cmd/mesh/install.go

```diff
@@ -182,6 +189,22 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 		return fmt.Errorf("failed to install manifests: %v", err)
 	}
 
+	// Workaround for broken validation with revisions (https://github.com/istio/istio/issues/28880)
+	// TODO(Monkeyanator) remove once we have a nicer solution
+	if iArgs.revision != "" {
```

Comment:
for external - can we check if the webhook points to `istiod` and create it, if not we will do nothing?

I think it will confuse users to have `istiod`

Reply:
Makes sense, changed to check for `istiod` in VWH